CONTRIBUTING.md

```diff
@@ -26,6 +26,7 @@ atrophius:
     bio: Here's a bio for me.
     url: http://www.jeremyprivett.com
     twitter: Atrophius
+    github: Atrophius
```

Comment:
I'd prefer explicit over implicit so that it's less likely someone will get confused by this not working. Also, omitting it here allows people to not link back to their Github if they decide.